catkin_tools/common.py

```diff
@@ -399,6 +400,19 @@ def log(*args, **kwargs):
             unicode_error_printed = True
 
 
+__warn_terminal_width_once_has_printed = False
+
+
+def __warn_terminal_width_once():
+    global __warn_terminal_width_once_has_printed
+    if __warn_terminal_width_once_has_printed:
+        return
+    __warn_terminal_width_once_has_printed = True
+    print('WARNING: Could not determine the width of the terminal. '
+          'This warning will only be printed once.',
+          file=sys.stderr)
```

Comment:
I prefer notice to warning anyway :-)
Speaking of noticing, I saw that only the OSX build is failing, maybe the terminal ioctl() does not work with that OS?